### PR TITLE
Document `AppleOps` and `MPSOps`

### PR DESCRIPTION
**Pull request by danieldk**
## Description

Document `AppleOps` and `MPSOps`.

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [x] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: website/docs/api-backends.md [hunk 1]
```diff
@@ -17,16 +17,18 @@ specialized versions can be called for different backends. You can also create
...
+| Backend    |        CPU         |        GPU         |        TPU        | Description                                                                                                 |
+| ---------- | :----------------: | :----------------: | :---------------: | ----------------------------------------------------------------------------------------------------------- |
+| `NumpyOps` | <i name="yes"></i> | <i name="no"></i>  | <i name="no"></i> | Execute via `numpy`, [`blis`](https://github.com/explosion/cython-blis) (optional) and custom Cython.       |
+| `AppleOps` | <i name="yes"></i> | <i name="no"></i>  | <i name="no"></i> | Use AMX matrix multiplication units on Apple Silicon Macs.                                                  |
+| `CupyOps`  | <i name="no"></i>  | <i name="yes"></i> | <i name="no"></i> | Execute via [`cupy`](https://cupy.chainer.org/) and custom CUDA.                                            |
+| `MPSOps`   | <i name="yes"></i> | <i name="yes"></i> | <i name="no"></i> | Use the GPU on Apple Silicon Macs for PyTorch models, use AMX matrix multiplication units for Thinc Models. |
```
Comment (svlandeg) [inline website/docs/api-backends.md:25]: Shall we sort these alphabetically?

File: website/docs/api-backends.md [hunk 2]
```diff
@@ -17,16 +17,18 @@ specialized versions can be called for different backends. You can also create
...
-| `NumpyOps` | <i name="yes"></i> | <i name="no"></i>  | <i name="no"></i> | Execute via `numpy`, [`blis`](https://github.com/explosion/cython-blis) (optional) and custom Cython. |
-| `CupyOps`  | <i name="no"></i>  | <i name="yes"></i> | <i name="no"></i> | Execute via [`cupy`](https://cupy.chainer.org/) and custom CUDA.                                      |
+| Backend    |        CPU         |        GPU         |        TPU        | Description                                                                                                 |
+| ---------- | :----------------: | :----------------: | :---------------: | ----------------------------------------------------------------------------------------------------------- |
+| `NumpyOps` | <i name="yes"></i> | <i name="no"></i>  | <i name="no"></i> | Execute via `numpy`, [`blis`](https://github.com/explosion/cython-blis) (optional) and custom Cython.       |
+| `AppleOps` | <i name="yes"></i> | <i name="no"></i>  | <i name="no"></i> | Use AMX matrix multiplication units on Apple Silicon Macs.                                                  |
```
Comment (svlandeg) [inline website/docs/api-backends.md:23]: For `AppleOps` and `MPSOps`, let's add a note that they're new from 9.0 onwards.

File: website/docs/api-backends.md [hunk 3]
```diff
@@ -56,7 +58,7 @@ use_ops(blis_ops)
 
 | Name          | Type         | Description                                                                              |
 | ------------- | ------------ | ---------------------------------------------------------------------------------------- |
-| `name`        | <tt>str</tt> | **Class attribute:** Backend name, `"numpy"` or `"cupy"`.                                |
+| `name`        | <tt>str</tt> | **Class attribute:** Backend name, `"numpy"`, `"apple"` `"cupy"` or `"mps"`.             |
```
Comment (svlandeg) [inline website/docs/api-backends.md:61]:
```suggestion
| `name`        | <tt>str</tt> | **Class attribute:** Backend name, `"numpy"`, `"apple"`, `"cupy"` or `"mps"`.            |
```

File: website/docs/api-backends.md [hunk 4]
```diff
@@ -1572,7 +1574,7 @@ with use_ops("cupy"):
 
 | Argument   | Type         | Description                                           |
 | ---------- | ------------ | ----------------------------------------------------- |
-| `ops`      | <tt>str</tt> | `"numpy"` or `"cupy"`.                                |
+| `ops`      | <tt>str</tt> | `"numpy"`, `"apple"`, `"cupy"` or `"mps"`.            |
```
Comment (svlandeg) [inline website/docs/api-backends.md:1577]:
```suggestion
| `ops`      | <tt>str</tt> | `"numpy"`, `"apple"`, `"cupy"`, or `"mps"`.           |
```

**Review by svlandeg — APPROVED** [review]
Looks good! Just some very minor nitpicks.

File: website/docs/api-backends.md (hunk 1 above)
Comment (danieldk) [reply website/docs/api-backends.md:25]: Done.

File: website/docs/api-backends.md (hunk 2 above)
Comment (danieldk) [reply website/docs/api-backends.md:23]: Added a note for `AppleOps`, `MPSOps` was already in 8.1.x.

File: website/docs/api-backends.md (hunk 4 above)
Comment (danieldk) [reply website/docs/api-backends.md:1577]: Ah, I thought we didn't do Oxford comma?

Comment (svlandeg) [reply website/docs/api-backends.md:1577]: Yea this suggestion was wrong, in the previous one the comma was missing between the second and third entity and my brain thought it was the same here, but it's not 😉

Comment (danieldk) [reply website/docs/api-backends.md:1577]: Fixed the other one 😁

**Pull request merged**